data/deep missions.txt

```diff
@@ -590,29 +605,27 @@ mission "Deep: Mystery Cubes 4"
 				"Corvette (Missile)" 2
 				
 	on enter Alnitak
-		dialog `Alnitak is in chaos when you enter the system. An outnumbered defense fleet is fighting a group of pirate ships, and among the pirates you spot three Cruisers.`
+		dialog `Alnitak is in chaos when you enter the system. An outnumbered defense fleet is fighting a group of pirate ships, and among the pirates you spot the three missing Cruisers.`
 
 	on visit
-		dialog `You have landed on <planet>, but there are still pirates attacking the system. Destroy all of the pirates in order to complete this mission.`
+		dialog `You have landed on <planet>, but there are still pirates attacking the system. Defeat all of the pirates in order to complete this mission.`
 	
 	on enter Hassaleh
 		set "deep: passed through north"
 	
 	on complete
 		payment 1000000
 		conversation
-			`As you come in for a landing, you can see that a number of buildings have taken fire from the pirate fleet above. White tents have been set up for tending to the wounded and multiple ships are in the process of being repaired. You are directed to a building that has sustained no damage in which a number of Navy officers, the admiral you met on Luna, Lieutenant Paris, and the remaining merchant captains who agreed to help have convened. After you enter the room and get settled down, the admiral begins to speak.`
+			`As you come in for a landing, you can see that a number of buildings have taken fire from the pirate fleet above. White tents have been set up for tending to the wounded and multiple ships are in the process of being repaired. You are directed to an unscathed building in which a number of Navy officers, the admiral you met on Luna, Lieutenant Paris, and your fellow merchant captains have convened. After you enter the room and get settled down, the admiral begins to speak.`
 			`	"Here's the sitrep. Beelzebub's ship was spotted entering the system, followed by the rest of the pirate fleet. His ship seemed to disappear in the confusion, so we have no idea where he went. Either Beelzebub is currently hiding on the anarchist worlds or made a dash for the Core, in which case the chase goes on."`
-			branch core
+			branch north
 				has "deep: passed through north"
```

Comment:
This needs to be changed to `not "deep: passed through north"`, as right now it's skipping over Paris' comments about going through the Far North if you actually did.